autopts/wid/gatt_client.py

```diff
@@ -760,6 +760,12 @@ def hdl_wid_58(params: WIDParams):
         logging.error("parsing error")
         return False
 
+    btp.clear_verify_values()
+
+    if params.test_case_name == "GATT/CL/GAR/BV-07-C":
+        btp.gatt_cl_read_long(btp.pts_addr_type_get(), btp.pts_addr_get(),
```

Comment:
This will result in two calls for test GATT/CL/GAR/BV-07. Is this what you wanted?

Reply:
Fixed